trainee_template/react_series.amlg

```diff
@@ -1265,8 +1327,7 @@
 				(if derivation_failed
 					(assign (assoc
 						series_data (trunc series_data)
-						influential_cases (list)
-						local_class_probabilities_map (assoc)
+						case_detail_values_map (map (lambda (call GetDetailDataStructure (assoc detail (current_index 1) ))) (zip requested_details))
```

Comment:
you need to put this map statement into GetDetailDataStructure, and have the method take requested_details as a parameter, so that the method outputs the full assoc directly.  that way this and all instances of this line are shortened down to:
`case_detail_values_map (call GetDetailDataStructure (assoc requested_details))`

Reply:
done!